docs/5.x/opscenter.md

```diff
@@ -1,73 +1,70 @@
-# Setting up an Ops Center
+# Introduction
 
-An Ops Center controls access and lifecycle of Applicaion Clusters and provides a distribution endpoint
-for Application Bundles to be installed on Gravity Clusters.
+The Gravity Ops Center is a multi-cluster control plane available in the Enterprise version of Gravity. It reducues the operational overhead of managing multiple Gravity Clusters and Applications Bundles by allowing users to:
 
-## Pre-requisites
+* Publish Application Bundles and manage their versions.
```

Comment:
Are we still calling this "Application Bundles", not "Cluster Images"? Will that terminology change come later?

Reply:
Yes, I think that is next update.